docs/snippets/quickstart-install.md

```diff
@@ -1,22 +1,60 @@
+!!! todo "Installing the `quickstart` plugin"
+    === "Using Homebrew"
+        For macOS, you can install the `quickstart` plugin by using <a href="https://github.com/knative/homebrew-client">Homebrew</a>.
```

Comment:
```suggestion
        For macOS, you can install the `quickstart` plugin by using [Homebrew](https://github.com/knative/homebrew-client)
```

Although actually shouldnt this really be a link to homebrew's website, not ours?

Reply:
I copied the link from the link in the `kn` section, but I can update to Hombrew proper if that's better

Reply:
Ahh, wait ... perhaps this should be https://github.com/knative-sandbox/homebrew-kn-plugins instead

Reply:
I just don't think it makes sense if the link text is "Homebrew" for the target not to be brew.sh. If the link is "using homebrew" that would be better at least (or we could just link to brew.sh, which I think is what I'd expect a link that says Homebrew to point to)